src/libraries/StackUint256.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.4.13;
+
+import "ROOT/libraries/DelegationTarget.sol";
+import "ROOT/legacy_reputation/Ownable.sol";
+
+contract StackUint256 is DelegationTarget, Ownable {
+    
+    uint256[] public collection;
+    uint256 public head;
+    address public owner;
+    bool public initialized;
```

Comment:
Yeah, this also made me realize I'll need to do the same thing on Orders.sol.  There's no perfect solution, unfortunately.